app/HutchAgent/Services/WorkflowTriggerService.cs

```diff
@@ -1,265 +1,55 @@
 using System.Diagnostics;
-using System.IO.Compression;
-using System.Text.Json.Nodes;
 using System.Text.RegularExpressions;
 using HutchAgent.Config;
 using HutchAgent.Constants;
-using HutchAgent.Data.Entities;
+using HutchAgent.Models;
 using Microsoft.Extensions.Options;
-using Microsoft.FeatureManagement;
 using ROCrates;
-using ROCrates.Exceptions;
-using ROCrates.Models;
-using File = System.IO.File;
 
 namespace HutchAgent.Services;
 
-[Obsolete]
 public class WorkflowTriggerService
 {
   private readonly WorkflowTriggerOptions _workflowOptions;
   private readonly ILogger<WorkflowTriggerService> _logger;
   private readonly string _activateVenv;
   private const string _bashCmd = "bash";
-  private readonly ROCrate _roCrate = new();
-  private readonly WorkflowJobService _WorkflowJobService;
-  private readonly IFeatureManager _featureManager;
   private readonly CrateService _crateService;
 
   public WorkflowTriggerService(
     IOptions<WorkflowTriggerOptions> workflowOptions,
     ILogger<WorkflowTriggerService> logger,
-    IServiceProvider serviceProvider,
-    IFeatureManager featureManager,
     CrateService crateService)
   {
     _logger = logger;
-    _featureManager = featureManager;
     _crateService = crateService;
     _workflowOptions = workflowOptions.Value;
     _activateVenv = "source " + _workflowOptions.VirtualEnvironmentPath;
-    _WorkflowJobService = serviceProvider.GetService<WorkflowJobService>() ?? throw new InvalidOperationException();
   }
 
   /// <summary>
-  /// Parse ROCrate metadata using the ROCrates library
+  /// Execute Wfexs given a job Id and input crate
```

Comment:
```suggestion
  /// Execute workflow given a job Id and input crate
```
Just because we're making everything less wfexs specific, naming-wise